components/organisms/Repositories/repositories.tsx

```diff
@@ -37,7 +37,7 @@ const Repositories = ({ repositories }: RepositoriesProps): JSX.Element => {
     isError: repoListIsError,
     isLoading: repoListIsLoading,
     setPage,
-  } = useRepositories(repositories, Number(range), Number(limit));
+  } = useRepositories(repositories, Number(range || "30"), Number(limit || "10"));
```

Comment:
refactor (non-blocking): it resolves to a number, but I think elsewhere we're using 30 and 10 in number form.

Reply:
Ok, I see where its being used in number form. Updated